lib/icedfrisby.js

```diff
@@ -626,36 +626,64 @@ class Frisby {
    * @return {object}
    * @desc HTTP header expect helper
    */
-  expectHeader (header, content) {
+  expectHeader (header, content, options) {
     const self = this
 
     header = (header + "").toLowerCase()
+    options = {
+      allowMultipleHeaders: ((options && options.hasOwnProperty('allowMultipleHeaders')) ? options.allowMultipleHeaders : false)
```

Comment:
Would `options.allowMultipleHeaders || false` be sufficient?

Reply:
Options can be null, hence the safety. We could create an empty object and safely deep merge, or something, but that doesn't feel nicer.

Reply:
Ah, I gotcha. This is really verbose though. I'd prefer something like
```js
options = options || {}
```
or, perhaps better
```js
options = Object.assign({ allowMultipleHeaders: false }, options)
```

Reply:
I agree with @paulmelnikow on this principal. Let's just use  a default parameter, which Node.js 6 allows:

```js
expectHeader (header, content, options = {}) {
  // ...
}
```

Reply:
Ace. With the default, we can use `options.allowMultipleHeaders || false`. That's much nicer.